web/themes/custom/pineapple/sass/layouts/nodes/_article.scss

```diff
@@ -0,0 +1,54 @@
+.article--teaser {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+  @include breakpoint($desk) {
+    width: 75%;
+    flex-direction: row;
+  }
+
+  .teaser-content {
+    background-color: #f1f0f4;
+    padding: 0 30px 30px 30px;
+    width: 100%;
+
+    @include breakpoint($desk) {
+      height: 400px;
+    }
+
+    p {
+      color: $primary;
+    }
+
+    .field--name-field-a-summary {
+      margin-top: 20px;
+      margin-bottom: 30px;
+    }
+
+    .teaser-link {
+      background-color: $primary;
+      padding: 15px;
+      margin-top: 20px;
+      width: 150px;
+      color: white;
```

Comment:
Restyle these things especially the "teaser-link" which is a button. Use the button model used on healthmove and the following link is always here on hand for extra guidance: https://www.uwl.ac.uk/themes/custom/rutherford/pattern-lab/public/?p=viewall-atoms-buttons

Reply:
thank you, done.